docs/custom-traces.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[custom-traces]]
+== Custom Traces
+
+If you want to track and time a custom event that happens in your application during a transaction,
+you can add a new trace to an existing transaction.
+
+This is an example of how to use custom traces.
+For general information about the Elastic APM Node.js Trace API,
+see the <<trace-api,Trace API documentation>>.
+
+In the example below we create an Express app that:
+
+1. Times how long it takes to receive the body of an HTTP POST or PUT request
+2. And times how long it takes to parse JSON sent by the client
```

Comment:
To make this info easier to scan, consolidate repeated info in the intro and say:

In the example below we create an Express app that times how long it takes to:

1. Receive the body of an HTTP POST or PUT request
2. Parse JSON sent by the client